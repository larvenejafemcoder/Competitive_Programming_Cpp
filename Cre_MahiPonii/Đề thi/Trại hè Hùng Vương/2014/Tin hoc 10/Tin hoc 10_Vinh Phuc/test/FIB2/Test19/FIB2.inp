16
18 1733
28 511042
27 114163
6 4
25 112665
21 7714
14 243
4 3
12 67
12 35
38 25722860
38 25650086
33 3067498
26 46245
24 63740
40 37058899
