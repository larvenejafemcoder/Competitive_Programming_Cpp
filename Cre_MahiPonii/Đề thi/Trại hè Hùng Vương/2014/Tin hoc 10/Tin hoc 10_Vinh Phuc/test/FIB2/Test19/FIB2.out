662
195201
43607
2
43034
2947
93
1
26
13
9825258
9797461
1171680
17664
24347
14155240
